feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -289,7 +282,8 @@ class BillingWrapper(
             return
         }
 
-        val alreadyAcknowledged = purchase is GooglePurchaseWrapper && purchase.containedPurchase.isAcknowledged
+        val originalGooglePurchase = purchase.originalGooglePurchase
+        val alreadyAcknowledged = originalGooglePurchase?.isAcknowledged ?: false
```

Comment:
If it's a restored purchase (what used to be a PurchaseHistoryRecordWrapper), `originalGooglePurchase` will be null, and we will just try to acknowledge again, which is the previous behavior (PurchaseHistoryRecord didn't provide any info regarding acknowledging so [we were always acknowledging](https://github.com/RevenueCat/purchases-android/blob/develop/purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt#L1280))

Reply:
I guess there's no damage in acknowledging a restored purchase, right?

Reply:
No, it will fail saying it's already acknowledged, nothing bad